components/monitoring/README.md

```diff
@@ -1,214 +1,233 @@
+A single
+[datasource (Thanos querier)](https://github.com/redhat-appstudio/infra-deployments/blob/16e48656370dc65dba6471a9f50d745832535723/components/monitoring/grafana/base/grafana-app.yaml#L216),
+`appstudio-datasource` is defined and it lets us query metrics from the Platform and
+User Workload Monitoring Prometheus.
 
+To use this default datasource any definition of a datasource in the dashboard json file
+should be removed or a `templating` should be used.
 
-### 4. Grafana dashboards
+## 4. Grafana Dashboards
 
 A dashboard is a set of one or more panels organized and arranged into one or more rows. 
 It visualizes results from multiple data sources simultaneously.
-New dashboards can be added through the user interface, preconfigured in the infra-deployments repository, 
-or imported from other projects.
-
-
-#### Create a dashboard
-
-  1. [Create a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/)
-  for your team's view of your service's Service Level Indicators.
-  (After navigating to your folder, + Create Dashboard)  
-  ***Note:***  
-  Creating a new dashboard manually is available only for development environment.  
-  You may copy and edit the `example` dashboard json instead, and test the new dashboard on the staging and production environments. 
-
-      The `example` dashboard json definition can be found 
-      [here](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/dashboards/generic-dashboards/example.json)
-
-
-  2. Add tiles to the dashboard to track your initial set of service level indicators.  
-  If the servicemonitor was added correctly to the stage Prometheus datasource, 
-  it will show up in the query list when you edit a tile.
-
-  3. Export the dashboard definition in JSON format. 
-  (At the top of the screen, the icon with 3 dots lets you "Share dashboard or panel". Select Export... Save to file.)
-   
-
-#### Team's repository
-
-Follow the next steps to define a dashboard in your team's repository
-
-  1. The dashboard should be located in the team’s repository, no change in `infra-deployments` is required,
-  the recommended structure is:
-      ```
-      ├── grafana
-      │   ├── dashboards
-      │   │   └── <teams_dashboard>.json
-      │   ├── <GrafanaDashboard resource file>
-      │   └── kustomization.yaml
-      ```
-      For example:
-      ```
-      ├── grafana
-      │   ├── dashboards
-      │   │   └── example-dashboard.json
-      │   ├── dashboard.yaml
-      │   └── kustomization.yaml
-      ```
-
-  2. Create a folder in your team's repository to maintain the dashboard configuration (e.g. grafana)
+New dashboards can be added through the user interface, preconfigured in the
+infra-deployments repository, or imported from other projects.
+
+### Create a Dashboard
+
+1. [Create a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/)
```

Comment:
I see that our Grafana version is already 10.4.3

```suggestion
1. [Create a dashboard](https://grafana.com/docs/grafana/v10.4/dashboards/)
```

Reply:
done